extensions/quicktime/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quicktime",
+  "title": "QuickTime",
+  "description": "This extension has collection of commands that lets you start screen recording, movie recording and sound recording instantly.",
+  "icon": "quicktime.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "start-screen-recording",
+      "title": "Start Screen Recording",
+      "subtitle": "QuickTime Player",
```

Comment:
I don't think Quick Time is involved here, it's the Screenshot utility to take care of the recording.

Reply:
as per the code I'm invoking QuickTime player's screen recording option which is same as from "screenshot" app. Based on that, I thought it might be appropriate to name the subtitle as QuickTime Player.